### DISCUSSION TITLE
建议！

### DISCUSSION THREAD

**Discussion by zjy903838822** [share121/inter-knot#2316] [其他]
希望绳网管理员给绳匠们一个改名的机会！！！
发现用的是github的注册名
![QQ20240816-011826](https://github.com/user-attachments/assets/7b022835-d290-441a-8b47-4094ecac9a05)

**Comment by Dave-12138** [comment]
github ID也是可以改的
不过现在显示在绳网上的是id。。。。其实应该显示名字的

**Reply by nostalgiatan** [reply]
这不找121改一下。

**Comment by Buer-Nahida** [comment]
https://github.com/settings/admin
点<kbd>Change username</kbd>

**Reply by zjy903838822** [reply]
这个还是会改的，但绳网上不显示啊

**Comment by zjy903838822** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/8e53cb4b90a9bbb67db92a9ca95cfd19.png" alt="我的心好痛" width="75" >

**Comment by zjy903838822** [comment]
谢谢绳匠们，我的委托委托解决了
![5338dd20-6cb8-4263-9dfb-5c5b3dced400](https://github.com/user-attachments/assets/20b8ccfc-0f57-4826-b30d-7cde933ded6d)
